EducationPortal.DAL/Repositories/FileStorageRepositories/FSUnitOfWork.cs

```diff
@@ -0,0 +1,111 @@
+﻿using EducationPortal.DAL.Entities;
+using EducationPortal.DAL.FS;
+using EducationPortal.DAL.Interfaces;
+
+namespace EducationPortal.DAL.Repositories.FileStorageRepositories
+{
+    public class FSUnitOfWork : IUnitOfWork
+    {
+        private readonly EducationPortalContext context;
+        
+        private GenericRepository<Course> courseRepository;
+        private GenericRepository<Article> articleRepository;
+        private GenericRepository<Video> videoRepository;
+        private GenericRepository<Book> bookRepository;
+        private GenericRepository<Profile> profileRepository;
+        private GenericRepository<Skill> skillRepository;
+        private GenericRepository<Role> roleRepository;
+
+        public FSUnitOfWork(EducationPortalContext educationPortalContext)
```

Comment:
it should be interface

Reply:
and inject all repositories

Reply:
Changed to interface.
Not sure what exactly i should did with repositories